var/spack/repos/builtin/packages/rocm-smi-lib/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+from os import popen
+
+
+class RocmSmiLib(CMakePackage):
+    """It is a C library for Linux that provides a user space interface
+       for applications to monitor and control GPU applications."""
+
+    homepage = "https://github.com/RadeonOpenCompute/rocm_smi_lib"
+    url      = "https://github.com/RadeonOpenCompute/rocm_smi_lib/archive/rocm-3.5.0.tar.gz"
+
+    maintainers = ['srekolam', 'arjun-raj-kuppala']
+
+    version('3.5.0', sha256='a5d2ec3570d018b60524f0e589c4917f03d26578443f94bde27a170c7bb21e6e')
+
+    variant('build_type', default='Release', values=("Release", "Debug"), description='CMake build type')
+    depends_on('cmake@3:', type='build')
+
+    @run_after('install')
+    def post_install(self):
+        popen('cp -R {0}/rocm_smi/lib {1}'.format(self.prefix, self.prefix))
+        popen('cp -R {0}/rocm_smi/include {1}'.format(self.prefix,
+                                                      self.prefix))
+        popen('rm -R {0}/rocm_smi'.format(self.prefix))
```

Comment:
These should use Spack and/or Python commands, not OS commands. We want Spack to someday support Windows, and there are a lot of differences in the flags that system commands use on macOS vs Linux.

Spack has a `copy_tree` and `install_tree` commands you can use to copy directories. As for deletion, I think the `os` module has some functions for that that work on any OS.